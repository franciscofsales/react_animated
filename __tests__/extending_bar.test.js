import 'react-native';
import React from 'react';
import {ExtendingBar} from '../';

import renderer from 'react-test-renderer';
it('renders correctly', () => {
  const tree = renderer.create(
    <ExtendingBar />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});