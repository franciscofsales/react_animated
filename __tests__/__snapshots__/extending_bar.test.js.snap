exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "transparent",
      "flex": 1,
      "flexDirection": "row",
    }
  }>
  <View
    style={
      Object {
        "backgroundColor": "transparent",
        "flexDirection": "column",
      }
    }>
    <View
      style={
        Object {
          "backgroundColor": "#de0e1b",
          "height": 8,
          "width": 0,
        }
      } />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "transparent",
        "flexDirection": "column",
      }
    }>
    <View
      style={
        Object {
          "backgroundColor": "#de0e1b",
          "height": 1,
          "width": 0,
        }
      } />
    <View
      style={
        Object {
          "backgroundColor": "transparent",
          "height": 6,
          "width": 0,
        }
      } />
    <View
      style={
        Object {
          "backgroundColor": "#de0e1b",
          "height": 1,
          "width": 0,
        }
      } />
  </View>
  <View
    style={
      Object {
        "backgroundColor": "transparent",
        "flexDirection": "column",
      }
    }>
    <View
      style={
        Object {
          "backgroundColor": "#de0e1b",
          "height": 8,
          "width": 0,
        }
      } />
  </View>
</View>
`;
